8.0/Dockerfile.fedora

```diff
@@ -0,0 +1,73 @@
+FROM registry.fedoraproject.org/f32/s2i-base:latest
```

Comment:
Moving to f35, we can use 8.0 in Fedora as well. It may be done in a separate PR though, as this one focuses on C9S.